src/js/media-time-range.js

```diff
@@ -445,7 +448,7 @@ class MediaTimeRange extends MediaChromeRange {
       && !this.mediaPaused
       && !this.mediaLoading
       && !this.mediaEnded
-      && this.mediaDuration > 0
+      && this.mediaSeekableEnd > 0
```

Comment:
NOTE: Although `mediaDuration` would probably be fine here, updating to `mediaSeekableEnd` since that's the value that's actually driving the updates under most cases.